### PR TITLE
Fixed Hebrew & Same Language Bugs

### PR DESCRIPTION
**Pull request by penwoodjon**
## What's New:
- Fixed bug with Hebrew translations not showing the correct language summary tag
- Made it so if input and output language is the same a result will not show up

### CODE REVIEW

File: integration.js [hunk 1]
```diff
@@ -95,54 +99,58 @@ function doLookup(entities, options, cb) {
...
-            data: {
-              ...result.body.data,
-              translations: [{
-                translatedText,
-                detectedSourceLanguage: sourceLanguage
-              }]
-            }
-          };
+          Logger.debug({ detectedSourceLanguage }, 'Translation Data');
+          const inputAndOutputLanguageIsSame = detectedSourceLanguage === options.outputLanguage.value;
```
Comment (sarus) [inline integration.js:103]: @penwoodjon I don't think we have two many people translating into Hebrew but do we need to normalize the `options.outputLanguage.value` for Hebrew.

Maybe the easiest fix here is to use `iw` as the language code in the config.js and config.json for Hebrew?

**Review by sarus — COMMENTED** [review]
@penwoodjon Quick question on ignoring when input and output language are the same for Hewbrew.

**Comment by sarus** [comment]
@penwoodjon In addition to previous comment I think this needs to merge into `develop` vice `master`

File: integration.js (hunk 1 above)
Comment (penwoodjon) [reply integration.js:103]: Changed in the config files!

**Review by sarus — APPROVED** [review]

**Pull request merged**